docs/conf.py

```diff
@@ -138,7 +138,7 @@ def get_tutorial_images():
 # General information about the project.
 project = 'MSS - Mission Support System'
 copyright = \
-    '2016-2023 by the MSS team, see AUTHORS, ' \
+    '2016-2024 by the MSS team, see AUTHORS. ' \
```

Comment:
The strings are concatenated on a single line and separated by different characters now. This seems odd. Should they all be separated by a period?